docs/README.md

```diff
@@ -112,6 +112,7 @@ For convenience, wildcards can be used: `pull_request.*`, `issues.*`, `pull_requ
     reviewers: [ user1, user2 ] # list of github usernames required to review
     owners: true # Optional boolean. When true, the file .github/CODEOWNER is read and owners made required reviewers
     assignees: true # Optional boolean. When true, PR assignees are made required reviewers.
+    pending_reviewer: true #Optional boolean. When true, all the reviewer's approval is 
```

Comment:
Missing a space after the `#` -- Also looks like an intentional unfinished sentence?

Reply:
fixed